app/Console/Commands/WishHappyBirthdayCommand.php

```diff
@@ -11,8 +11,12 @@ class WishHappyBirthdayCommand extends Command
 {
     protected $signature = 'spatie:happy-birthday';
 
+    protected int $ageFilter = 40;
```

Comment:
😂

Reply:
No need to face the harsh reality on your birthday 😬

Reply:
haha